test/test_helper.exs

```diff
@@ -1 +1,3 @@
+{_, 0} = System.cmd("epmd", ["-daemon"])
+Node.start(:"ex_unit@127.0.0.1", :longnames)
```

Comment:
Would be helpful to leave a comment above these two clarify the exact purpose of using this for future readers.